itou/templates/dashboard/dashboard.html

```diff
@@ -319,6 +319,12 @@ <h5 class="card-header">Admin</h5>
                                 Export des candidatures
                             </a>
                         </li>
+                        <li class="card-text mb-3">
+                            <i class="ri-external-link-line ri-lg ri-lg"></i>
```

Comment:
Petite classe en double :)
```suggestion
                            <i class="ri-external-link-line ri-lg"></i>
```
---
Je pense que l'icone `external-link` devrait être à droite du lien plutôt qu'à gauche, c'est globalement comme ça que je l'ai vu dans les maquettes jusqu'à présent et dans le cas actuel ça rendrais plus explicite que ce lien là est différent des autres.

Mais pour ne pas dépareiller il faudrait surement trouver une autre icone pour ne pas avoir un trou à gauche, je n'ai pas l'impression qu'on ai une icone spécifique pour les PASS IAE, je n'ai trouvé que cette image : 
![logo_pass_iae](https://raw.githubusercontent.com/betagouv/itou/master/itou/static/img/pass_iae/logo_pass_iae.svg)

Reply:
💯 !